### PR TITLE
:sparkles: Add blog post about sealed secrets and helm

### PR DESCRIPTION
**Pull request by ernesgonzalez33**

### CODE REVIEW

File: content/posts/secrets-charts.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+* [Homebrew](https://github.com/bitnami-labs/sealed-secrets#homebrew)
+* [MacPorts](https://github.com/bitnami-labs/sealed-secrets#macports)
+* [Nixpkgs](https://github.com/bitnami-labs/sealed-secrets#nixpkgs)
+* [Linux](https://github.com/bitnami-labs/sealed-secrets#linux)
+* [From Source](https://github.com/bitnami-labs/sealed-secrets#installation-from-source)
+
+Then, run the following command: 
+
+```bash
+echo -n <my-secret> | kubeseal --controller-name=sealed-secrets --controller-namespace=sealed-secrets --raw --from-file=/dev/stdin --scope=namespace-wide
```
Comment (alvarolop) [inline content/posts/secrets-charts.md:77]: One thing: When you say <my-secret>, you mean the secret content, not the full k8s definition. I'm not sure if this is 100% clear. Maybe you could add an example?

Comment (ernesgonzalez33) [reply content/posts/secrets-charts.md:77]: You are right! I will add more on that part

Comment (ernesgonzalez33) [reply content/posts/secrets-charts.md:77]: @alvarolop check it now!

Comment (alvarolop) [reply content/posts/secrets-charts.md:77]: LGTM now!

**Review by alvarolop — APPROVED** [review]
LGTM

**Pull request merged**